pkg/unstructuredutil/yaml.go

```diff
@@ -0,0 +1,11 @@
+package unstructuredutil
```

Comment:
why not having it part of the `pkg/controller` ?

Reply:
Because `unstructured.Unstructured` is not something exclusive to controllers or the `controller-runtime`, so it can potentially be used in other places of our codebase